modules/openapi-generator/src/main/resources/go-server/controller-api.mustache

```diff
@@ -235,4 +239,28 @@ func (c *{{classname}}Controller) {{nickname}}(w http.ResponseWriter, r *http.Re
 	// If no error, encode the body and the result code
 	EncodeJSONResponse(result.Body, &result.Code,{{#addResponseHeaders}} result.Headers,{{/addResponseHeaders}} w)
 
-}{{/operation}}{{/operations}}
+}
+
+{{#allParams}}{{#isBodyParam}}
```

Comment:
Is there a case where a body param is not a schema object?

If multiple body param references the same schema object, this will be duplicated repeatedly. 

Could this be moved into the model.mustache?